backend/pkg/database/resource.go

```diff
@@ -520,3 +524,9 @@ func (r *ResourceTableHandler) setCountFilters(tx *gorm.DB, filters *CountFilter
 
 	return tx, nil
 }
+
+func (r *ResourceTableHandler) setViewRefreshHandlerIfExists() {
+	if r.viewRefreshHandler != nil {
+		r.viewRefreshHandler.SetTrue()
+	}
+}
```

Comment:
if we change the code so that the views can register refresh handlers at the point they are created, then we can always have a viewRefreshHandler, and have createAllViews only register refresh funcs if DB type == postgres, or in the future maybe some other DB type?